jaeger/charts/linkerd-jaeger/templates/jaeger-injector.yaml

```diff
@@ -37,6 +37,7 @@ spec:
       {{- include "linkerd.tolerations" (dict "Values" .Values.webhook) | nindent 6 }}
       {{- end -}}
       {{- include "linkerd.node-selector" (dict "Values" .Values.webhook) | nindent 6 }}
+      {{- include "linkerd.affinity" (dict "Values" .Values.webhook) | nindent 6 }}
```

Comment:
When `nodeAffinity` is not set, the output is including an empty line. I think you need to wrap this to check for the existance of `.Values.webhook.affinity` (the same applies to the other templates).

Reply:
That make sense, will work on it, thanks.